plasmapy/particles/particle_class.py

```diff
@@ -2087,7 +2105,7 @@ def __init__(self, mass: u.kg = None, charge: (u.C, Real) = None, symbol=None):
                 f"{mass} and a charge of {charge}."
             ) from exc
 
-    def __repr__(self) -> str:
+    def __repr__(self):
```

Comment:
I'm confused about this removal of type hinting. Wasn't it fine before?
```suggestion
    def __repr__(self) -> str:
```

Reply:
It was.  I was thinking that `__repr__` methods will always return strings so the type hint was redundant, but I'm happy to put it back since explicit is better than implicit.